account_db_mapper.py

```diff
@@ -0,0 +1,15 @@
+class AccountDatabaseMapper:
+
+    def __init__(self,db):
+        self.db = db
+
+    def save(self,account):
+        self.db.cursor.execute(
+            "INSERT INTO accounts "+
+            "(email,google_access_token,google_refresh_token) " +
+            "VALUES (%s,%s,%s)",
+            (account.email,account.access_token,account.refresh_token)
+        )
+
```

Comment:
I understand the what the tokens are, my point is that do we want to continuously access google API or would be better served by getting the info we want from the calendar once and then storing that in the database. Since the refresh tokens expire when user revokes access we might have to keep adding and removing them from the DB even during a single use session. Again, for 1.0 we might just have to accept this behaviour.